app/components/Settings/Scheduler.react.js

```diff
@@ -0,0 +1,262 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import ReactDataGrid from 'react-data-grid';
+import ms from 'ms';
+import matchSorter from 'match-sorter';
+import Highlight from 'react-highlight';
+
+import { Link } from '../Link.react';
+import { Row, Column } from '../Layout.react';
+import Modal from '../Modal.react';
+import { plotlyUrl } from '../../utils/utils';
+
+const NO_OP = () => {};
+
+export const SQL = (props) => <Highlight className="sql">{props.children}</Highlight>;
+SQL.propTypes = {
+  children: PropTypes.string
+};
+
+class QueryFormatter extends React.Component {
+  static propTypes = {
+    value: PropTypes.object
+  }
+
+  render() {
+    const query = this.props.value;
+    return (
+      <Row>
+        <Column style={{ paddingRight: '24px', fontSize: 15 }}>
+          <SQL>{query.query}</SQL>
+        </Column>
+      </Row>
+    );
+  }
+}
+
+class IntervalFormatter extends React.Component {
+  static propTypes = {
+    value: PropTypes.object
+  }
+
+  render() {
+    const run = this.props.value;
+    return (
+      <Row>
+        <Column style={{ paddingRight: '24px' }}>
+          <em
+            style={{
+              fontSize: 15
+            }}
+          >
+            {`Runs every ${ms(run.refreshInterval * 1000, { long: true })}`}
+          </em>
+        </Column>
+      </Row>
+    );
+  }
+}
+
+const rowStyle = {
+  justifyContent: 'flex-start',
+  borderBottom: '1px solid rgba(0, 0, 0, 0.05)',
+  padding: '16px 0px'
+};
+const boxStyle = { boxSizing: 'border-box', width: '50%' };
+
+const MetaPreview = props => {
```

Comment:
There is a styling issue with this component. See:

![screenshot from 2018-07-02 17-11-19](https://user-images.githubusercontent.com/6199391/42174693-1e2aea9e-7e1b-11e8-868d-debf9f2912bf.png)